STEER/STEER/AliGenerator.cxx

```diff
@@ -450,19 +450,36 @@ void AliGenerator:: SetHighWaterMark(Int_t nt)
   //
   if (fStack)
      fStack->SetHighWaterMark(nt);
-  else 
+  else
      gAlice->GetMCApp()->SetHighWaterMark(nt);
-   
+
 }
 void AliGenerator::FinishRun()
 {
     ;
 }
 
+///-------------------------------------------------------------------------
+Bool_t AliGenerator:: ApplyUserTrigger()
+{
+  // User Trigger
+  if(fSetUserTrig){
+    if(!fUserTrigger){
+      AliFatal("No external trigger defined!");
+      return kTRUE;
+    }
+    else{
+      if(!fUserTrigger(fStack)){return kFALSE;}
+      else return kTRUE;
+    }
+  }
+}
```

Comment:
Ciao Luca,
Sorry, my messages were not clear probably, I think that you removed again the last return... This is needed. Just don't put "else return kFALSE" (or kTRUE, i don't remember), but simply "return kFALS" (or kTRUE).

Reply:
Sorry, I misunderstood! Now it is fixed

Reply:
Thanks!